site/docsource/OCaml-call-JS.adoc

```diff
@@ -1268,11 +1268,11 @@ var v2 = {
 Compiler infers the type of `v2` as below:
 [source,ocaml]
 --------------
-val v2 : object
-  method hi : float -> float -> float
-  method say : float -> float
-  method x : unit -> float
-end [@bs]
+val v2 : <
+  hi : float -> float -> float;
+  say : float -> float;
+  x : unit -> float
+> [@bs]
```

Comment:
Right, of course. So would it be ok if I just add the `[@bs.meth]` attributes, or should I revert it? I think it's a bit misleading to pass it off as valid syntax, but I could add a warning instead: "Compiler infers the type of `v2` as below (in pseudo-syntax)"